docs/DEVICE-CONNECTIVITY.md

```diff
@@ -137,3 +137,239 @@ The device reports the status of all of the device connectivity using [SystemAda
+ the published netdumps. To limit the overhead associated with tracing and packet capture,
+ NIM is only allowed to enable them and produce netdump at most once per hour *before onboarding*
+ and once per day *after onboarding* ([configurable](./CONFIG-PROPERTIES.md)
+ by `netdump.topic.preonboard.interval` and `netdump.topic.postonboard.interval`, respectively).
+ The pre-onboard interval is intentionally lower (by default) to get more frequent diagnostics
+ for initial connectivity troubleshooting.
+- `/config` and `/info` requests done by zedagent to obtain device configuration and publish info
+ messages, respectively. Packet capture is not enabled in this case. Netdump is produced at the interval
+ as given by `netdump.topic.postonboard.interval` (`/config` and `/info` requests are not run before
+ onboarding). For `/info` requests, tracing only covers publication of the `ZInfoDevice` message.
+ Moreover, tracing is enabled only if the highest priority DPC is currently being applied
+ and is reported by NIM as working. Otherwise, we will eventually get `nim-fail*` netdump
+ which should be sufficient for connectivity troubleshooting. The purpose of zedagent netdumps
+ is to debug issues specific to `/config` an `/info` requests (which are essential to keep the device
+ remotely manageable).
+ Netdumps are published separately into topics `zedagent-config-<ok|fail>`
+ and `zedagent-info-<ok|fail>`.
+- *every* download request performed by [downloader](../pkg/pillar/cmd/downloader) using the HTTP
+ protocol is traced and netdump is published into the topic `downloader-ok` or `downloader-fail`,
+ depending on the outcome of the download process. By default, this does not include any packet
+ captures. However, a limited PCAP can be enabled with config option `netdump.downloader.with.pcap`.
+ Limited in the sense that it will not include TCP segments carrying non-empty payload (i.e. packets
+ with the downloaded data). On the other hand, included will be all UDP traffic (DNS requests),
+ TCP SYN, RST and FIN packets as well as ACK packets without data piggybacking (sufficient to tell
+ where a download process got "broken"). The total PCAP size is limited to 64MB (packets past this
+ limit will not be included).
+
+In order to troubleshoot a present or a past connectivity issue, it is necessary to locate and obtain
+the appropriate netdump from the affected device (locate by microservice aka topic name and look for
+the closest timestamp). Without a remote connectivity to the device, it is possible to dump all
+diagnostics to a USB stick. See [CONFIG.md](./CONFIG.md), section "Creating USB sticks".
+With this method, the entire `/persist/netdump` directory will be copied over.
+If device is remotely accessible, published netdumps can be listed and copied over ssh (if enabled
+by config), edgeview (`ls` + `cp` commands; to download all netdumps at once use: `tar//persist/netdump`)
```

Comment:
add tar command to list here

Reply:
It is mentioned after (...; to download all netdumps at once use: `tar//persist/netdump`)